Adadie/Fresh
# 26-store-queries-posts-on-firestore

**What does this PR do?**
This PR will link new post form and contact form to Firestore database by adding firebase SDKs and scripts for saving and retrieving data from the database.

**Description of Task to be completed?**
Send data from forms to Firestore.
Retrieve data from Firestore to the respective section in the profiles page.

**How should this be manually tested?**
Clone this repository in your local directory [link](https://github.com/Junior-Adadie/Fresh.git).
Open contact.html with your browser.
Fill out the contact form and inspect with the console.
Comment out the codes for saving contact form messages.
Open profile.html with your browser.
Inspect the page with the console, there should be no error.
Make sure your browser is updated.

**Any more context you want to provide?**
When data from post form is sent, it doesn't work, it returns "Cannot read property 'addEventListener' of null at Firestore.js:18" from the console but when the function for sending data from the contact form is removed, the rest of the actions work just fine both sending and retrieving.

**What are the relevant Trello stories?**
[Trello Card](https://trello.com/c/53oZ2eEy/26-store-queries-posts-on-firestore)
[Trello Card 2](https://trello.com/c/S9Gqai9U/36-functions-for-sending-data-are-conflicting)

**Screenshots (if appropriate)**

![Console error](https://user-images.githubusercontent.com/68109866/89712266-733f3500-d998-11ea-9e9b-f8f9218269b0.PNG)
![post](https://user-images.githubusercontent.com/68109866/89712268-79351600-d998-11ea-8ac1-9dd87653379a.PNG)
